### PR TITLE
Support tabs as whitespace

### PR DESCRIPTION
**Pull request by joshprice**
See https://github.com/elixir-makeup/makeup_erlang/pull/29 for more info

### CODE REVIEW

File: lib/makeup/lexers/elixir_lexer.ex [hunk 1]
```diff
@@ -25,7 +25,7 @@ defmodule Makeup.Lexers.ElixirLexer do
   # TODO: check we're following this convention
   # NOTE: if Elixir had a good static type system it would help us do the right thing here.
 
-  whitespace = ascii_string([?\r, ?\s, ?\n, ?\f], min: 1) |> token(:whitespace)
+  whitespace = ascii_string([?\r, ?\s, ?\n, ?\f, ?\t], min: 1) |> token(:whitespace)
```
Comment (josevalim) [inline lib/makeup/lexers/elixir_lexer.ex:28]: We don’t allow \f as white space though, so we can remove it.

```suggestion
  whitespace = ascii_string([?\r, ?\s, ?\n, ?\t], min: 1) |> token(:whitespace)
```

Comment (joshprice) [reply lib/makeup/lexers/elixir_lexer.ex:28]: Sure can remove it. What about Erlang? Both already supported `?/f`

Comment (josevalim) [reply lib/makeup/lexers/elixir_lexer.ex:28]: I checked \f in Erlang before merging there and it worked!

Comment (joshprice) [reply lib/makeup/lexers/elixir_lexer.ex]: Ok removed

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: lib/makeup/lexers/elixir_lexer.ex (hunk 1 above)
Comment (joshprice) [reply lib/makeup/lexers/elixir_lexer.ex]: Sorry commit message messed up as I keep forgetting I can't use backticks in commit msgs on the command line, please squash

**Pull request merged**